hashintel/hash
# H-2473, H-2584: Add new HASH app homepage screens

## 🌟 What is the purpose of this PR?

Adds new homepage screens (internal [designs](https://www.figma.com/design/gydVGka9FjNEg9E2STwhi2/HASH?node-id=11634-6408&t=9Kvph0gBWEHFrelI-4)) for:
- logged out users
- logged in users who are on the waitlist
- logged in users who have access

## 🚫 Blocked by




- [x] Merging [the PR](https://github.com/hashintel/internal/pull/152) (internal) which adds the 'get waitlist' functionality
  - technically the new screens can be added without this PR, they just won't show a waitlist position and will fall back to 'You are on the waitlist'
  
## 🔍 What does this change?




As well as adding the screens:
- Consolidates some of our modal styles where modals have a header and close button
- H-2584: Create an `@example` organization in hosted HASH (app.hash.ai, locally)

## Pre-Merge Checklist 🚀

### 🚢 Has this modified a publishable library?




This PR:


### 📜 Does this require a change to the docs?




The changes in this PR:


### 🕸️ Does this require a change to the Turbo Graph?




The changes in this PR:


## 🛡 What tests cover this?



- Some existing

## ❓ How to test this?



1. Check out the homepage as a logged out user, logged in user
2. To test waitlist:
  - run the internal API off the branch linked above
  - in the HASH `.env.local` set an allow list:
```
USER_EMAIL_ALLOW_LIST=["alice@example.com", "bob@example.com", "admin@example.com"]
```
  - sign up with an email that is already on the HASH list (e.g. your own that is signed up to app.hash.ai). first request will take some time as the internal API needs to build its cache of the list
  - try the early access form

## 📹 Demo



#### Logged out -> Sign up, on waitlist -> sign in to user with access

https://github.com/hashintel/hash/assets/37743469/bc99c557-97fc-4cc5-9485-ac44e2baa972



#### User not on allow list, self-hosted instance

<img width="1677" alt="Screenshot 2024-05-22 at 12 19 28" src="https://github.com/hashintel/hash/assets/37743469/dfbe00ea-2cf6-42ef-a49e-8ab1c6297107">
